internal/pkg/plugin/jenkins/jenkins.go

```diff
@@ -23,24 +23,26 @@ var defaultHelmConfig = helm.Options{
 	},
 }
 
-// getHelmResourceAndCustomResource wraps helm resource and custom resource,
-// this is due to the limitation of `plugininstaller`,
-// now `plugininstaller.GetStateOperation` only support one resource get function,
-// if we want to use both existing resource get function(such as helm's methods) and custom function,
-// we have to wrap them into one function.
-func getHelmResourceAndCustomResource(options plugininstaller.RawOptions) (map[string]interface{}, error) {
-	// 1. get helm resource
-	resource, err := helm.GetPluginAllState(options)
+func genJenkinsState(options plugininstaller.RawOptions) (statemanager.ResourceState, error) {
+	resState, err := helm.GetPluginAllState(options)
 	if err != nil {
 		return nil, err
 	}
 
-	outputs := map[string]interface{}{}
+	// svc_name.svc_ns:svc_port
+	outputs := map[string]interface{}{
+		"jenkins_url": "http://jenkins.jenkins:8080",
```

Comment:
note: service name could be changed if the user changes the default configuration.

Reply:
> note: service name could be changed if the user changes the default configuration.

I know it. But users can't change the svc name from values.yaml and it's no meaning.

Reply:
The svc name of Jenkins is determined by chart release name.

<img width="878" alt="image" src="https://user-images.githubusercontent.com/36830265/186103879-bbc74c69-f246-4528-b33b-286e026d3978.png">

Reply:
```go
func (opts *jenkinsOptions) getJenkinsFullName() string {
	if strings.Contains(opts.Chart.ChartName, opts.Chart.ReleaseName) {
		return opts.Chart.ReleaseName
	}

	return fmt.Sprintf("%s-%s", opts.Chart.ReleaseName, opts.Chart.ChartName)
}
```

Reply:
get it!

Reply:
I'll fix it in the next pr. Thanks for the input!